src/openrct2-ui/windows/RideConstruction.cpp

```diff
@@ -3422,16 +3422,18 @@ void ride_construction_toolupdate_entrance_exit(const ScreenCoordsXY& screenCoor
 void ride_construction_tooldown_construct(const ScreenCoordsXY& screenCoords)
 {
     const CursorState* state = context_get_cursor_state();
-    RideId rideIndex;
-    int32_t trackType, trackDirection, liftHillAndAlternativeState, z, properties, highestZ;
+
     rct_window* w;
 
     map_invalidate_map_selection_tiles();
     ride_construction_invalidate_current_track();
 
     CoordsXYZ mapCoords{};
+    RideId rideIndex;
+    int32_t trackType, trackDirection, liftHillAndAlternativeState, z, highestZ;
+
     if (window_ride_construction_update_state(
-            &trackType, &trackDirection, &rideIndex, &liftHillAndAlternativeState, &mapCoords, &properties))
+            &trackType, &trackDirection, &rideIndex, &liftHillAndAlternativeState, &mapCoords, nullptr))
```

Comment:
https://github.com/OpenRCT2/OpenRCT2/search?q=window_ride_construction_update_state

Looks like all calls to this function pass `nullptr` now. You can drop this argument entirely.

Reply:
This was marked as resolved, but it isn't resolved. The last argument is still there.